### PR TITLE
Added more information about choosing query planner

### PR DESCRIPTION
**Pull request by systay**

### CODE REVIEW

File: content/en/docs/13.0/reference/compatibility/query_planner.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+ + v3
+   - This is the default planner. It is not being worked on anymore - new features are added to gen4. 
+
+ + gen4
+   - The new planner added to vitess. 
+   It supports more queries, and is able to find plans that are more optimal than the old planner could.
+
+ + left2right
+   - This planner works similarly to the `straight_join` directive in MySQL - the joins are going to be performed in the order specified by the query.
```
Comment (GuptaManan100) [inline content/en/docs/13.0/reference/compatibility/query_planner.md]: `similarly` -> `similar`

File: content/en/docs/13.0/reference/compatibility/query_planner.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Since this is a critical piece of the system, we have made it easy to change which planner is used.
+
+The following planners are available:
+
+ + v3
+   - This is the default planner. It is not being worked on anymore - new features are added to gen4. 
+
+ + gen4
+   - The new planner added to vitess. 
+   It supports more queries, and is able to find plans that are more optimal than the old planner could.
```
Comment (frouioui) [inline content/en/docs/13.0/reference/compatibility/query_planner.md:17]: we could link the gen4 blog post here

File: content/en/docs/13.0/reference/compatibility/query_planner.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+ + left2right
+   - This planner works similarly to the `straight_join` directive in MySQL - the joins are going to be performed in the order specified by the query.
+   
+ + gen4fallback
+   - This planner will first try the gen4 planner, and if it fails, it uses the V3 planner. This was useful during the development of gen4, but probably not very useful in production.
```
Comment (frouioui) [inline content/en/docs/13.0/reference/compatibility/query_planner.md]: nit: `V3` -> `v3`

**Review by frouioui — APPROVED** [review]

File: content/en/docs/13.0/reference/compatibility/query_planner.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Example of using the query hint to specify planner:
+
+```sql
+select /*vt+ PLANNER=gen4 */ * from commerce;
+```
+
+### Testing the new planner using `gen4comparev3`
+
+The latest version of the Vitess query planner, `Gen4`, changes most of the planner's internals.
+To ensure `Gen4` produces plans that, once executed, give us the same results as what `V3` would do, we have developed a small test tool that runs queries using both `Gen4` and `V3` planners and compare their results.
```
Comment (GuptaManan100) [inline content/en/docs/13.0/reference/compatibility/query_planner.md]: ```give us the same results as what `V3` would do``` -> give us the same results as what `V3` would ~~do~~

Comment (GuptaManan100) [inline content/en/docs/13.0/reference/compatibility/query_planner.md]: compares their results

**Review by GuptaManan100 — APPROVED** [review]

**Pull request merged**